src/stylesheets/_input_fields.scss

```diff
@@ -212,7 +213,7 @@
             transform: translate(-50%, 0) rotate(45deg);
             top: -6px;
             width: 1rem;
-            content: '';
+            content: "";
```

Comment:
Single quotes is more in line with what we did other places.

Reply:
If those other places are SCSS then that will get updated accordingly…

Reply:
They are not. We should be as consistent cross-language when possible. Arbitrarily making a change like this that is going to be at odds with my muscle memory.